client/state/themes/selectors/is-full-site-editing-theme.ts

```diff
@@ -20,6 +20,10 @@ export function isFullSiteEditingTheme(
 		return false;
 	}
 
+	if ( theme.block_theme ) {
+		return true;
+	}
+
 	const themeFeatures = getThemeTaxonomySlugs( theme, 'theme_feature' );
 
 	return themeFeatures.includes( 'full-site-editing' );
```

Comment:
Can we actually drop this check, so that Calypso is 100% free from this tag? I believe, Live Preview is the only thing that's still depending on this tag. Curious why.

Reply:
I left it for safety in the short term if the `block_theme` attribute is not there for some reason. I guess we can simply return `theme.block_theme` and get rid of this tag in the future.

Reply:
Sure, just wanted to make sure that we're aware 🙏